tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -126,7 +126,7 @@ suite('Language server validation', async () => {
         } catch (err) {
             Logger.debug('Workaround for the https://github.com/eclipse/che/issues/18974.');
             await browserTabsUtil.refreshPage();
-            await ide.waitAndSwitchToIdeFrame();
+            // await ide.waitAndSwitchToIdeFrame(); ??? Why is it still here? How did the test not fail?
```

Comment:
We have not been running old Happy path tests for a several months, because they don't support Dev Workspaces and so are absolute.

There is DevWorkspaceHappyPath test running instead.

Reply:
updated